### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ErvinSabic**
Updating README to use beta version so the tutorial works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ and go to [http://localhost:4200](http://localhost:4200).
 Installing the library is as easy as:
 
 ```bash
-ember install ember-simple-auth
+npm i ember-simple-auth@3.1.0-beta.1
```
Comment (sdebarros) [inline README.md:104]:
```suggestion
ember install ember-simple-auth@3.1.0-beta.1
```

**Review by sdebarros — COMMENTED** [review]
Thanks for your MR! I left one correction.

**Comment by ErvinSabic** [comment]
> Thanks for your MR! I left one correction.

Added correction to use ember install instead of npm.

**Review by sdebarros — APPROVED** [review]
Thanks!

**Pull request merged**